Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -1833,6 +1837,8 @@ protected function forgetClass($className): void
 
         unset($this->classReflectionData[$className]);
         unset($this->classesCurrentlyBeingForgotten[$className]);
+
+        $this->removedReflectionDataClasses[$className] = true;
```

Comment:
todo check if `forgetClass` is also called in other occasions and if this flag makes sense in that case.